ergotree-ir/src/mir/value.rs

```diff
@@ -158,6 +159,8 @@ pub enum Value {
     Tup(TupleItems<Value>),
     /// Transaction(and blockchain) context info
     Context,
+    /// Block header
+    Header(Header),
```

Comment:
Due to the Rust memory allocation for enum (always allocate by the biggest enum variant size) it needs to be boxed.